mspray/apps/main/tests/views/test_performance.py

```diff
@@ -34,3 +43,190 @@ def test_team_leader_performance(self):
         response = view(request, slug=sprayarea.id)
         self.assertEqual(response.status_code, 200)
         self.assertTrue('Supervisor' in str(response.render().content))
+
+    def test_mda_spray_operator_summary_view(self):
+        """Test MDASprayOperatorSummaryView."""
+        data_setup()
+        self._load_fixtures()
+        rhc = Location.objects.get(name='Chadiza_104')
+        spray_operator = SprayOperator.objects.first()
+        team_leader = TeamLeader.objects.first()
+        spray_operator.team_leader = team_leader
+        spray_operator.rhc = rhc
+        spray_operator.save()
+
+        spray_day = SprayDay.objects.filter(spray_operator=spray_operator)
+        spray_day.update(sprayable=True)
+
+        performance_report(spray_operator)
+        report1 = PerformanceReport.objects.get(spray_operator=spray_operator)
+        report1.found = 7
+        report1.reported_sprayed = 16
+        report1.reported_found = 22
+        report1.save()
+
+        report2 = PerformanceReport.objects.get(spray_operator=spray_operator)
+        report2.id = None
```

Comment:
I think we can make this a tiny bit better - it hard for someone reading the code to figure out why we need report2 and report1 and why/if they are different.  Could you add some comments that explain this?

Reply:
cool...let me include some comments to explain the test flow